azure-spring-boot/src/main/java/com/microsoft/azure/keyvault/spring/Constants.java

```diff
@@ -7,10 +7,13 @@
 package com.microsoft.azure.keyvault.spring;
 
 public class Constants {
+    // TODO (wepa) Load service version dynamically
+    public static final String AZURE_KEYVAULT_SPRING_NAME = "Spring.Boot.Key.Vault/0.2.2-SNAPSHOT";
```

Comment:
keyvault-spring-boot-starter/0.2.2-SNAPSHOT, to align with docdb and package name